docker-compose/keycloak-realm/ups-realm-sample.json

```diff
@@ -59,7 +59,7 @@
             "enabled": true,
             "publicClient": true,
             "webOrigins": [
-                "*"
+                "http://localhost:9999"

```

Comment:
So this might be a weird thought, but would it be possible to make this configurable?  I understand it is a sample and if the user is running this in a evaluation/localhost environment then it probably won't be a problem.

Also this is a strange default for the port.

Reply:
that's what comes w/ the docker-compose file - it's machting ;-)

Reply:
Yeah I understand.  I'm fine with that.